htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -1038,15 +1038,26 @@ public function fetchAllByAccount($sortorder = '', $sortfield = '', $limit = 0,
 	 * @param string 		$sortfield                      Sort field
 	 * @param int 			$limit                          Limit
 	 * @param int 			$offset                         Offset limit
-	 * @param string|array 	$filter                         Filter array
+	 * @param array			$filter       					Filter as an Universal Search string.
```

Comment:
To make the transition with $filter that is a string, we must introduce the string here.
array will be abandonned after several versions of transitions.